src/css/_mixins.scss

```diff
@@ -0,0 +1,25 @@
+@mixin flexBox ($direction, $justify: none, $align: none){
```

Comment:
Nice use of the default arguments. I forgot about this in the last project and had to figure out the defaults to pass as arguments in each situation.